Внедрены Unit-тесты для контроллеров и сервисов.

timurash opened the pull request:
https://github.com/timurash/timeTrackingService/issues/15

timurash committed:
Внедрены Unit-тесты для контроллеров и сервисов.

AlexandrYZ commented:
Тесты добавлены. Но будет крайне полезно задокументировать это в описании проекта Readme.md. В описании следует указать команду которой можно запустить выполнение тестов в консоли -  `dotnet test --list-tests`

AlexandrYZ reviewed (commented):
Не указана задача по которой это потребовалось сделать.

AlexandrYZ commented on sources/TimeTrackingServiceTests/ReportControllerTests.cs:
```diff
@@ -0,0 +1,165 @@
+        }
+
+        [Fact]
+        public void UpdateReportWithModelErrorReturnsBadRequest()
```
Что тестирует этот метод?
Необходимо убедиться, что контроллер вернет ошибку валидации если была передана модель с ошибками.

timurash committed:
Исправлены 4 теста, которые проверяли случай невалидного ModelState

Теперь тесты осуществляют проверку, возвращает ли контроллер ошибку валидации

https://github.com/timurash/timeTrackingService/pull/14#discussion_r474075280
